tekton/cronjobs/README.md

```diff
@@ -1,93 +1,107 @@
 # Tekton Cron Jobs
 
 This folder holds kustomize overlays, that are used to maintain cron job
-configurations. To add a new cron job to be deployed to the `dogfooding`
-cluster, create a folder and add a `kustomization.yaml` into it, along with the
-cronjob overlay.
-
-There are three base cron jobs available:
-* `nightly-image-build-cron-base` which can be used to build container images
-  on a regular basis and push them to a container repo (by default
+configurations. To add a new cron job to be deployed to the a cluster,
```

Comment:
nit: `to the a cluster` -> `to a cluster`

Reply:
done